pkgs/by-name/ch/chemacs2/package.nix

```diff
@@ -0,0 +1,48 @@
+{
+  lib,
+  fetchFromGitHub,
+  stdenvNoCC,
+  unstableGitUpdater,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "chemacs2";
```

Comment:
Is the package called `chemacs` or `chemacs2`? If the later, can you fix the commit?

Reply:
`Chemacs` 1 is the old and deprecated package. 
I will fix this.